lib/sailor.js

```diff
@@ -60,41 +60,44 @@ class Sailor {
 
     startup() {
         return co(function* doStartup() {
-            console.log('Starting up component');
+            log.info('hook-startup begin');
             const result = yield this.invokeModuleFunction('startup');
-
+            log.debug('hook-startup result', { result });
             const handle = hooksData.startup(this.settings);
             try {
                 const state = _.isEmpty(result) ? {} : result;
                 yield handle.create(state);
             } catch (e) {
                 if (e.statusCode === 409) {
-                    console.warn('Hooks data creating conflict. Deleting existing data');
+                    log.warn('hook-startup – hooks data creating conflict. Deleting existing data');
                     yield handle.delete();
                     yield handle.create(result);
                 } else {
+                    log.warn('hook-startup – error');
```

Comment:
It's reported, general error handler doesn't tell suggestion where exactly error has occurred (e.g. when stacktrace point to somewhere in request.js etc).